pkg/terraform/install.go

```diff
@@ -28,8 +30,25 @@ func (m *Mixin) Install() error {
 		step.Flags = append(step.Flags, builder.NewFlag("input=false"))
 	}
 
-	for _, k := range sortKeys(step.Vars) {
-		step.Flags = append(step.Flags, builder.NewFlag("var", fmt.Sprintf("'%s=%s'", k, step.Vars[k])))
+	if step.CreateVarFile {
+		vf, err := m.FileSystem.Fs.Create(path.Join(m.WorkingDir, m.TerraformVarsFilename))
```

Comment:
nit: this could be written as `m.FileSystem.Create(path.Join(m.WorkingDir, m.TerraformVarsFilename))`